frontend/client/src/styles/_loading.scss

```diff
@@ -34,6 +34,20 @@ $loading-offset: 25px;
 		width: 90%;
 		max-width: 1200px;
 	}
+
```

Comment:
Also could you remove the `TODO` comment above, since you have now resolved that issue?